Prace_domowe/Praca_domowa3/Grupa1/Chylak_Maciej/homework_3.ipynb

```diff
@@ -0,0 +1,692 @@
+{
```

Comment:
<p>Dobrze, że zastosowano kroswalidację.</p><p>Należy pamiętać aby oprócz mean podać też std.</p><p> </p><p>Uwaga na przyszłość:</p><ul><li>dot. wielokrotnego wywołania funkcji cross_val_score - w tym wypadku trening z kroswalidacją wykonano wiele razy tylko po to aby wyliczyć statystyki, z uwagi na losowość dzielenia danych w ramach kroswalidacji miary nie są wyliczane na takich samych zbiorach danych. Dlatego też lepiej byłoby zamiast cross_val_score użyć funkcji cross_validate, która zwraca wartość więcej niż jednej metryki.</li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/167/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>